CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2014-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.13.3)
```

Comment:
Yep, it's needed on Windows, to have the ability to create symlinks from CMake with its own command, it resolves an issue when VS2017 and VS2109 are installed together, it provides "target_link_options" command which is a massive improvement in how link flags are added.

In general this is a non-issue because you can download a pre-compiled binary from the CMake website, or compile it from scratch (which not difficult at all).